src/deepsparse/image_classification/schemas.py

```diff
@@ -53,8 +55,48 @@ def from_files(cls, files: List[str], **kwargs) -> "ImageClassificationInput":
             )
         return cls(images=files)
 
+    @staticmethod
+    def create_test_inputs(
```

Comment:
Why are we putting tests and test files in the source code? These should be migrated, along with the test files, to tests directory

Reply:
Thanks for the feedback @markurtz all test related code + files has been moved to test directories